README.md

```diff
@@ -113,6 +126,10 @@ The `--help` flag shows usage examples for the CLI.
 
 The `--info` will print the current version of the CLI.
 
+#### `--run`
```

Comment:
This command seems broken? I need to investigate a little further

Reply:
I decided to remove the documentation for now. Seems to have worked when we installed packages by default, which is no longer the case (probably to allow folks to BYO package manager w/o introducing a settings file for a default?)